paddle/fluid/pybind/pybind.cc

```diff
@@ -181,6 +184,53 @@ bool SupportsBfloat16FastPerformance() {
 #endif
 }
 
+std::tuple<std::unordered_set<std::string>, std::unordered_set<std::string>,
+           std::unordered_set<std::string>>
+OpSupportedInfos(std::string place, framework::proto::VarType::Type dtype) {
```

Comment:
Would you mind input a `const std::string& place` ?

This is just following C++ style:
https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

If the `std::toupper` causes the trouble for using const reference, you can do a copy inside the function.

Reply:
Done.